core/spec/models/spree/order_populator_spec.rb

```diff
@@ -18,5 +18,18 @@
         subject.populate(2, 5)
       end
     end
+
+    context 'with an invalid variant' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.